docs/Questions.md

```diff
@@ -0,0 +1,59 @@
+<H1> Folder structure(Pages, Components, Templates, Organisms, Molecules, Atoms) </H1>
+We should stick to one way of organizing our folder and component's structure.
+Utils have files with .tsx file extension, which I think is inappropriate.
+We have validation.js and wdyr.js files at the top level of src
+
+<H1> Styles </H1>
+Currently we both use Styled-Components and scss across our codebase. I think we should stick to one and remove the other
+
+<H1> Data fetching & selectors in components </H1>
+We have a lot of ambiguity about how we interact with data in our app.
+For example:
+
+- `useConnects` in components with different ways of solving the issue, when there is no needed data(like a venue id)
+- firestore direct pushes in `actions` and components
+
+<H1> Merging flow </H1>
+We should have a well-defined merging flow to avoid misunderstandings.
+
+<H1> Branch and commit naming </H1>
+Well-defined rules for this can help keep everything tidy and understandable by all parties
+
+<H1> Merging vs rebasing </H1>
+
+<H1> Casing for different types of files(Service, Component, Util functions, Styles, Types) </H1>
+We have different casings for the same file types. For example:
+
+- types/utility
+- types/forms
+- types/Role
+- types/Question
+
+<H1> Not related \ unused files and folders should be removed </H1>
+
+---
+
+<H3> Replace js with ts. At least `src` folder should not contain js files </H3>
```

Comment:
Agreed. I believe the `.js` files that exist are only there for legacy reasons, and us not having the time to clean them up yet.